designer_v2/lib/features/dashboard/studies_table.dart

```diff
@@ -31,142 +67,168 @@ class StudiesTable extends StatelessWidget {
-          sortable: true,
-        ),
-        StandardTableColumn(
-          label: tr.studies_list_header_created_at,
-          columnWidth: const FlexColumnWidth(1),
-          sortable: true,
-        ),
-        StandardTableColumn(
-          label: tr.studies_list_header_participants_enrolled,
-          columnWidth: MaxColumnWidth(FixedColumnWidth(statsCellWidth), const IntrinsicColumnWidth()),
-          sortable: true,
-        ),
-        StandardTableColumn(
-          label: tr.studies_list_header_participants_active,
-          columnWidth: MaxColumnWidth(FixedColumnWidth(statsCellWidth), const IntrinsicColumnWidth()),
-          sortable: true,
-        ),
-        StandardTableColumn(
-          label: tr.studies_list_header_participants_completed,
-          columnWidth: MaxColumnWidth(FixedColumnWidth(statsCellWidth), const IntrinsicColumnWidth()),
-          sortable: true,
-        ),
-      ],
-      onSelectItem: onSelect,
-      trailingActionsAt: (item, _) => getActions(item),
-      buildCellsAt: _buildRow,
-      sortColumnPredicates: _sortColumns,
-      pinnedPredicates: pinnedPredicates,
-      emptyWidget: emptyWidget,
-    );
-  }
+    if (studies.isEmpty) {
+      return emptyWidget;
+    }
 
-  int Function(Study a, Study b) get pinnedPredicates {
-    return (Study a, Study b) {
-      if (pinnedStudies.contains(a.id)) {
-        return -1;
-      } else if (pinnedStudies.contains(b.id)) {
-        return 1;
-      }
-      return 0;
-    };
-  }
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final isCompact = constraints.maxWidth < compactWidthThreshold;
+        final isSuperCompact = constraints.maxWidth < superCompactWidthThreshold;
+        final isCompactStatTitle = constraints.maxWidth < compactStatTitleThreshold;
+        debugPrint('studies table width: ${constraints.maxWidth}');
+        // Calculate the minimum stat column width
+        final int maxStatTitleLength =
+            isCompactStatTitle ? "Completed".length : tr.studies_list_header_participants_completed.length;
+        final double statsColumnWidth = maxStatTitleLength * 9.9;
 
-  List<int Function(Study a, Study b)?> get _sortColumns {
-    final predicates = [
-      (Study a, Study b) => 0, // do not sort pin icon
-      (Study a, Study b) => a.title!.compareTo(b.title!),
-      (Study a, Study b) => a.status.index.compareTo(b.status.index),
-      (Study a, Study b) => a.participation.index.compareTo(b.participation.index),
-      (Study a, Study b) => a.createdAt!.compareTo(b.createdAt!),
-      (Study a, Study b) => a.participantCount.compareTo(b.participantCount),
-      (Study a, Study b) => a.activeSubjectCount.compareTo(b.activeSubjectCount),
-      (Study a, Study b) => a.endedCount.compareTo(b.endedCount),
-    ];
-    return predicates;
-  }
+        // Calculate the minimum status column width
+        int maxStatusLength = "Entwurf".length;
```

Comment:
Should this be localized?

Reply:
No, it is a design choice what this lenght should be. The string just gives a hint why the specific length was chosen.